frontend/src/components/group/index.vue

```diff
@@ -1,83 +1,74 @@
-                            {{ $t('commons.button.delete') }}
-                        </el-button>
-                        <el-button link v-if="row.edit" type="primary" @click="search()">
-                            {{ $t('commons.button.cancel') }}
-                        </el-button>
-                        <el-button
-                            link
-                            v-if="!row.edit && !row.isDefault && !row.isDelete"
-                            type="primary"
-                            @click="setDefault(row)"
-                        >
-                            {{ $t('website.setDefault') }}
-                        </el-button>
-                    </div>
-                </template>
-            </el-table-column>
-        </ComplexTable>
-    </el-drawer>
+                <el-table-column :label="$t('commons.table.operate')">
+                    <template #default="{ row, $index }">
+                        <div>
+                            <el-button link v-if="row.edit" type="primary" @click="saveGroup(groupForm, row)">
+                                {{ $t('commons.button.save') }}
+                            </el-button>
+                            <el-button link v-if="!row.edit" type="primary" @click="editGroup($index)">
+                                {{ $t('commons.button.edit') }}
+                            </el-button>
+                            <el-button
+                                link
+                                v-if="!row.edit"
+                                :disabled="row.isDefault"
+                                type="primary"
+                                @click="deleteGroup($index)"
+                            >
+                                {{ $t('commons.button.delete') }}
+                            </el-button>
+                            <el-button link v-if="row.edit" type="primary" @click="search()">
+                                {{ $t('commons.button.cancel') }}
+                            </el-button>
+                            <el-button
+                                link
+                                v-if="!row.edit && !row.isDefault && !row.isDelete"
+                                type="primary"
+                                @click="setDefault(row)"
+                            >
+                                {{ $t('website.setDefault') }}
+                            </el-button>
+                        </div>
+                    </template>
+                </el-table-column>
+            </ComplexTable>
+        </template>
+    </DrawerPro>
 </template>
 <script lang="ts" setup>
 import { ref } from 'vue';
 import i18n from '@/lang';
 import { CreateGroup, DeleteGroup, GetGroupList, UpdateGroup } from '@/api/modules/group';
-import Header from '@/components/drawer-header/index.vue';
 import { MsgSuccess } from '@/utils/message';
 import { Group } from '@/api/interface/group';
 import { Rules } from '@/global/form-rules';
```

Comment:
There are no major bugs or potential issues found in the provided code snippet. However, there may be some small improvements that can be made in terms of readability and maintainability:

1. Use consistent formatting: Make sure to follow a standard coding convention consistently throughout the code.

2. Use more descriptive variable names: Variables should have clear and meaningful naming conventions which make it easier for other developers (and you yourself) to understand what they represent.

3. Add logging functionality: This is quite handy if something goes wrong with an operation. Logging is often included into error handling mechanisms and might help when diagnosing issues on debugging sessions or production setups where logs will not reach users directly but give clues about what went wrong.

4. Avoid magic strings/variables (like '$t()'s), replace them with expressions where applicable:
```
export interface CommonProps {
  t?: () => string,
}
```